src/containers/Contactus.jsx

```diff
@@ -0,0 +1,230 @@
+import { Formik, Form, Field, ErrorMessage } from 'formik';
+import { useRef } from 'react';
+import emailjs from 'emailjs-com';
+import contactus from '../assets/contactus.svg';
+
+function Contactus() {
+  const modelRef = useRef();
+
+  const sendEmail = (values) => {
```

Comment:
Can you try to move the sendEmail function to a src/utils/utils.js file and import it here ? That way the code would look cleaner. If it's not possible since the function has lots of inbuilt emailJS parameters inside, just let me know.